contrib/tools/common.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash -eExl
+
+WORKSPACE=${WORKSPACE:=$PWD}
+ucx_inst=${WORKSPACE}/install
+CUDA_MODULE="dev/cuda11.1.1"
+GDRCOPY_MODULE="dev/gdrcopy2.1_cuda11.1.1"
```

Comment:
1. shall we remove these from test_jenkins.sh and reuse this file?
2. maybe move this to buildlib/tools, this way all CI related will will be in buildlib